test/Makefile

```diff
@@ -1,8 +1,5 @@
 .PHONY: test_binaries test
 
-CFLAGS := 
-LDFLAGS :=
-
```

Comment:
> a) The whole idea of disabling them in test was to make sure things actually worked without any special flags being used. (i.e. that the shared library was not dependent on the compiler version or compilation flags used)
b)

In Debian we can't distinguish regular compilations from test compilations, so it is easier for us to patch in the flag propagation everywhere. I'll remove this part from this PR